LibraryBookRegistration/Book.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibraryBookRegistration
+{
+    /// <summary>
+    /// Represents a customer
```

Comment:
Documentation for Book at line 10: I believe should be "Represents a book" not "Represents a customer".

Reply:
Thank you very much!